dist/apache-unomi-tracker.cjs.js

```diff
@@ -0,0 +1,1571 @@
+'use strict';
```

Comment:
Why having cjs, esm, umd files instead of only one minified file?

Reply:
Authoring libraries is always a pain if you try to maximize the number of projects that can incorporate yours. 

In a perfect world, we would have to use only one module system, most probably ES modules as that is the standard in Javascript specifications, but as we might know we don't live in a perfect world and transitioning projects takes a lot of time. So, even though now, NodeJS already has pretty good support for ES modules and all modern browsers are compatible with it a lot of projects still use CJS or UMD or even AMD modules.

Reply:
And minified process should be done by the projects using the library directly, most of them are using bundler (WebPack, rollup, etc) and are already managing minification on there side.
When it come to have a real JavaScript library, minification should not be done on the library directly.
Our target at this point is not the browser usage directly, devs will need to package this lib in their own bundle system.